packages/web-components/fast-foundation/src/disclosure/disclosure.ts

```diff
@@ -0,0 +1,145 @@
+import { attr, FASTElement } from "@microsoft/fast-element";
+/**
+ * A Disclosure Custom HTML Element.
+ * Based largely on the {@link https://w3c.github.io/aria-practices/#disclosure | disclosure element }.
+ *
+ * @public
+ */
+export class Disclosure extends FASTElement {
+    /**
+     * Determines if the element should show the extra content or not.
+     *
+     */
+    @attr({ mode: "boolean" })
+    public expanded: boolean;
```

Comment:
Do you need an expandedChanged handler to respond to attribute changes?  I think you'd need to call update(), for example.